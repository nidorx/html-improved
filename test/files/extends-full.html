<extends file="./includes/page.html" />

<vars
    page-title="Html Improved Sample"
    otherVar="value on 'index.html'"
/>

<block name="content">
    <div> page content </div>
</block>

<block name="footer">
    <div> footer content </div>
</block>

<block name="footer" append>
    <div> after footer </div>
</block>

<block name="footer" prepend>
    <div> before footer </div>
</block>

