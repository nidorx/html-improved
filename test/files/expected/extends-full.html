<!DOCTYPE html>
<html>
    <head>
        <title>Html Improved Sample</title>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0"> </head>

    <body>
        <h1>value on &#39;index.html&#39;</h1>
        <div>
            <ul>
                <li> Home - Application name </li>
                <li> #1 - value one </li>
                <li> #2 - value two </li>
            </ul>
            <div> navigation content </div>
        </div>
        <div>
            <div> page content </div>
        </div>
        <div>
            <div> before footer </div>
            <div> footer content </div>
            <div> after footer </div>
        </div>
    </body>

</html>